Resolutions/ChildOfLight_Resolution/rules.txt

```diff
@@ -108,26 +107,28 @@ formats = 0x01a
 overwriteWidth = ($width/$gameWidth) * 512
 overwriteHeight = ($height/$gameHeight) * 512
 
-[TextureRedefine] # Half-res
+[TextureRedefine] # Half-Res
+width = 640
+height = 368
+tileModesExcluded = 0x001
+overwriteWidth = ($width/$gameWidth) * 640
+overwriteHeight = ($height/$gameHeight) * 368
+
+[TextureRedefine]
 width = 640
 height = 360
-formats = 0x001
```

Comment:
Defining a format prevents unwanted textures being upscaled and makes it easier to work on in the future since you keep track of what each TextureRedefine does.

Reply:
I am aware of this, but I didn't notice any visual bugs when removing this, so I thought it was unnecessary to keep this line. Slashiee doesn't seem to add formats in his packs unless it's really necessary. Perhaps a # could be added in front?

Reply:
It's more work to define it while not always being that necessary. Unless it helps the game in some way, I don't see why you would remove it.